ast/Trees.h

```diff
@@ -6,6 +6,7 @@
 #include "core/LocalVariable.h"
 #include "core/SymbolRef.h"
 #include "core/Types.h"
+#include <cassert>
```

Comment:
What is it for?

Reply:
I think it's a relic from when I was trying to use an assertion in a context where `ENFORCE` wouldn't work. I'll remove it.